src/main/java/com/postsquad/scoup/web/user/auth/request/AccessTokenRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.postsquad.scoup.web.user.auth.request;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class AccessTokenRequest {
+
+    private final String clientId;
+    private final String clientSecret;
+    private final String code;
```

Comment:
```suggestion
    private final String clientId;
    
    private final String clientSecret;
    
    private final String code;
```

DTO는 이런 식으로 하기로 했던 것 같아요

Reply:
앗 저는 몰랐는데 그렇게 하기로 했군요! 컨벤션에 추가해놓았습니다~ 혹시 엔티티와 DTO 사이에 줄바꿈 규칙을 다르게 가져가는 이유가 있을까요? 상의했었는데 제가 놓친 거라면 죄송해요😥

Reply:
저번에 이노가 이렇게 하셔서 습관됐다고 했던 것 같은데
제가 잘못 알아들었나봐요 🤦‍♂️
어떻게 하는게 좋을까요?
그리고 저는 엔티티도 저렇게 작성했던 것 같네요
어노테이션이 많아서 그렇기도 했지만

Reply:
앗 그건 클래스랑 첫번째 필드 사이에 한 줄 띄는거 말씀드린 거였어요😂

Reply:
그럼 깔끔하게 다 띄는 걸로해요😁

Reply:
네 좋습니다 @min27604 괜찮으신가요?

Reply:
네 저는 좋습니다